migrations/0033_payment_skus.up.sql

```diff
@@ -0,0 +1,3 @@
+alter table order_items add column payment_methods text[];
```

Comment:
I don't want to cause a major refactor and I might not know as much of the architecture, but wouldn't it make sense for there to be a `payments` table that joins with the `orders` table?

Reply:
I would actually suggest instead that this is moved to be a property of the order and called `allowed_payment_methods` to reflect the fact that it is defining potential payment methods which will be accepted.

I think it makes little sense to have it on each item as you cannot pay for an individual item, only the entire order.

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/816/commits/6276de6cc2ffc85e725c37c13ded51bd438902a1 payment method is pulled from all the order items, and attached to the order instead.